src/Tempest/Console/src/Actions/RenderConsoleCommand.php

```diff
@@ -18,8 +18,8 @@ public function __invoke(ConsoleCommand $consoleCommand): void
     {
         $parts = ["<em><strong>{$consoleCommand->getName()}</strong></em>"];
 
-        foreach ($consoleCommand->handler->getParameters() as $parameter) {
-            $parts[] = $this->renderParameter($parameter);
+        foreach ($consoleCommand->getArgumentDefinitions() as $argument) {
```

Comment:
What confuses me: where do you map the custom name to the actual input provided in the console? Eg. `./tempest my:command --custom-input=abc`

Reply:
The mapping is already done with the `ConsoleArgumentDefinition` so this is build using the `fromParameter()` method, then I've updated the name constructuon to give the attribute name ( added in this PR ) instead of the parameter and still fallback to it if not set.

So here we need to update the input from parameter to the argument definition (  which still can fallback to it )

Tell me if it's not clear 😅